docs/docs/architecture/cert-manager.md

```diff
@@ -0,0 +1,10 @@
+# Cert-manager
+
+Constellation ships with cert-manager preinstalled.
+The default installation is part of the `kube-system` namespace, as all other Constellation-managed components.
+You are free to install more instances of `cert-manager` into other namespaces.
+However, please be aware that any new installation need to use the same version as the one installed with Constellation.
```

Comment:
We could use a `:::caution` or `:::info` box to highlight this information.

Reply:
Couldn't find any information on this in the official docs. But I think there is nothing to be done apart from setting the namespace.